### PR TITLE
TomDoc methods in step.rb #274

### PR DESCRIPTION
**Pull request by bbiehl**
- [ ] video_time_conversion_for_steps
- [ ] get_draft_title_for_step

### CODE REVIEW

File: app/models/step.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@ class Step < ActiveRecord::Base
   }
   validates_presence_of :title, :chapter_id, :start_time, :end_time
 
+  # Internal:
```
Comment (sumeetjain) [inline app/models/step.rb]: Trailing whitespace detected.

**Comment by bbiehl** [comment]
ran rubocop...
99 files inspected, 388 offenses detected
whoa Nelly!

**Comment by bbiehl** [comment]
On second thought, I'll merge to master and redo if necessary.

File: app/models/step.rb [hunk 2]
```diff
@@ -6,13 +6,19 @@ class Step < ActiveRecord::Base
...
+  # Returns a String
   def video_time_conversion_for_steps
     start_time = Time.at(self.start_time).utc.strftime("%H:%M:%S")
     end_time = Time.at(self.end_time).utc.strftime("%H:%M:%S")
 
     return "#{start_time} - #{end_time}"
   end
 
+  # Gets draft video step title if step is a draft.
+  #
```
Comment (sumeetjain) [inline app/models/step.rb:20]: Trailing whitespace detected.

**Pull request merged**